src/load/setup.py

```diff
@@ -10,7 +10,7 @@
 
 
 # HISTORY.rst entry.
-VERSION = '0.3.4'
+VERSION = '1.0.0b1'
```

Comment:
I think you should set the version to `1.0.0` as your API version is stable.

Reply:
Nice catch. Thank you!